.github/workflows/build-ci.yml

```diff
@@ -41,7 +41,6 @@ jobs:
                     - '4.4'
                     - '5.0'
                 php:
-                    - '8.0'
```

Comment:
New laravel doesn't work with php 8.0?  Why you delete 8.0 from matrix in CI?

Reply:
https://github.com/laravel/framework/blob/10.x/composer.json#L18
I suggest to include if condition in ci
If laravel version >= 10.0 then on php 8.0 ci step will be disabled

Reply:
```
- laravel/framework 10.x-dev requires php ^8.1 -> your php version (8.0.27) does not satisfy that requirement.
- illuminate/support 10.x-dev requires php ^8.1 -> your php version (8.0.27) does not satisfy that requirement.
```